jumanji/wrappers.py

```diff
@@ -360,18 +360,40 @@ def render(self, state: State) -> Any:
         return super().render(state_0)
 
 
+OBS_IN_EXTRAS_KEY = "real_next_obs"
+
+
+def _obs_in_extras(
```

Comment:
Not sure if this is the best place for the constant/method, but it's used only by the auto reset wrappers, so thought it made sense to put it above them :smile:

Reply:
Having it here seems fine to me.